big_tests/tests/gdpr_SUITE.erl

```diff
@@ -210,6 +237,26 @@ retrieve_vcard(Config) ->
               Alice, Config, "vcard", ExpectedHeader, ExpectedItems)
         end).
 
+remove_vcard(Config) ->
+    escalus:fresh_story(Config, [{alice, 1}], fun(Alice) ->
+        AliceFields = [{<<"FN">>, <<"Alice">>}, {<<"LN">>, <<"Ecila">>}],
+        AliceSetResultStanza
+            = escalus:send_and_wait(Alice, escalus_stanza:vcard_update(AliceFields)),
+        escalus:assert(is_iq_result, AliceSetResultStanza),
+        AliceU = escalus_utils:jid_to_lower(escalus_client:username(Alice)),
+        AliceS = escalus_utils:jid_to_lower(escalus_client:server(Alice)),
+
+        maybe_stop_and_unload_module(mod_vcard, mod_vcard_backend, Config),
+        {0, _} = unregister(Alice, Config),
+
+        mongoose_helper:wait_until(
+            fun() ->
+                mongoose_helper:successful_rpc(mod_vcard, get_personal_data,
+                    [AliceU, AliceS])
+            end, [{vcard,["jid","vcard"],[]}])
```

Comment:
I don't think we really need to match on schema in this test case.

Reply:
The point here is that actually match on empty result of GDPR data. The schema is just because current check function requires it as argument